### PR TITLE
Added documentation and tests for Azure network and minor changes in the get Azure Network usages method

### PR DESCRIPTION
**Pull request by ganeshjagdale9545**

### CODE REVIEW

File: cartography/data/indexes.cypher [hunk 1]
```diff
@@ -226,5 +226,4 @@ CREATE INDEX ON :AzureNetworkRoute(id);
 CREATE INDEX ON :AzureNetworkSecurityGroup(id);
 CREATE INDEX ON :AzureNetworkSecurityRule(id);
 CREATE INDEX ON :AzurePublicIPAddress(id);
-CREATE INDEX ON :AzureLocation(id);
-CREATE INDEX ON :AzureUsege(id);
\ No newline at end of file
+CREATE INDEX ON :AzureNetworkUsege(id);
```
Comment (mpurusottamc) [inline cartography/data/indexes.cypher:229]: Spelling mistake. It should be `AzureNetworkUsage`

File: cartography/data/jobs/cleanup/azure_import_networks_usages_cleanup.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
+    "statements": [
+        {
+            "query": "MATCH (n:AzureNetworkUsege)<-[:CONTAIN]-(:AzureNetwork)<-[:RESOURCE]-(:AzureSubscription{id: {AZURE_SUBSCRIPTION_ID}}) WHERE n.lastupdated <> {UPDATE_TAG} WITH n LIMIT {LIMIT_SIZE} DETACH DELETE (n) return COUNT(*) as TotalCompleted",
```
Comment (mpurusottamc) [inline cartography/data/jobs/cleanup/azure_import_networks_usages_cleanup.json:4]: Please change `AzureNetworkUsege` to `AzureNetworkUsage` everywhere

**Review by mpurusottamc — CHANGES_REQUESTED** [review]
one minor change needed

**Review by mpurusottamc — APPROVED** [review]
Looks good. Thanks for making the changes.

**Comment by ganeshjagdale9545** [comment]
New Branch created for Azure Networks(azure-networks)

**Pull request closed without merging**